observationtree/src/main/scala/explore/observationtree/ObsBadge.scala

```diff
@@ -1,23 +1,30 @@
 // Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
 // For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
 
-package explore.components
+package explore.observationtree
 
 import cats.syntax.all._
+import explore.Icons
 import explore.components.ui.ExploreStyles
 import explore.model.ObsSummary
 import explore.model.reusability._
 import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.core.model.Observation
 import lucuma.core.util.Enumerated
 import react.common._
+import react.common.implicits._
+import react.semanticui.elements.button.Button
+import react.semanticui.sizes._
 import react.semanticui.views.card._
 
 final case class ObsBadge(
   obs:      ObsSummary,
   layout:   ObsBadge.Layout,
-  selected: Boolean = false
+  selected: Boolean = false,
+  deleteCB: Observation.Id => Callback
```

Comment:
Would it be OK to expose `IO` instead of `Callback`? Just to use the same style throughout.

Reply:
I thought about that and I was tempting to do `id => F[Unit]` rather than `IO` but as we know type params on `Props` require some hacks downstream.
I guess fixing to IO would be fine

Reply:
Slightly related, I left out `deleteCB` from `Reusability` but it should be possible using the `Reuse` mechanism.
Would that work on `IO` as well?

Reply:
If that's `scalajs-react`'s `Reusable`, then I would expect it to work, since it's generic. However, I experimented with this approach a bit and found it added complexity at the call site, since we have to build the `Reusable`. In this case at least, since the delete function won't really change, I think it's enough to leave out the function out of the reusability.